scrapers_next/va/committees_csv.py

```diff
@@ -0,0 +1,37 @@
+from paramiko.client import SSHClient, AutoAddPolicy
+import paramiko
+import time
+from spatula import CsvListPage
+
+
+class ChamberList(CsvListPage):
+    client = SSHClient()
+    client.set_missing_host_key_policy(AutoAddPolicy)
+    connected = False
+    attempts = 0
+    while not connected:
+        try:
+            client.connect(
+                "sftp.dlas.virginia.gov",
+                username="rjohnson",
+                password="E8Tmg%9Dn!e6dp",
```

Comment:
I realize this password is likely already exposed elsewhere, but can we pull the password from an environment variable instead?

Reply:
Something like `os.environ["VA_SFTP_PASSWORD"]` or something would be appropriate.